srml/support/procedural/src/storage/genesis_config/genesis_config_def.rs

```diff
@@ -114,27 +114,26 @@ impl GenesisConfigDef {
 				.unwrap_or_else(|| quote!( Default::default() ));
 
 			config_field_defs.push(GenesisConfigFieldDef {
-				doc: line.doc_attrs.clone(),
 				name: config_field,
 				typ,
+				attrs: line.doc_attrs.clone(),
 				default,
 			});
 		}
 
 		for line in &def.extra_genesis_config_lines {
-			let doc = line.attrs.iter()
-				.filter_map(|a| a.parse_meta().ok())
-				.filter(|m| m.name() == "doc")
+			let attrs = line.attrs.iter()
+				.map(|a| a.parse_meta().expect("attribute cannot be parsed"))
```

Comment:
First rule of the macro club, never panic ;)

If we want to accept all attributes here, we don't need to parse them.

Just store `syn::Attribute` in `attrs`.

So:
`let attrs = line.attrs.clone()`